src/app/qgisapp.cpp

```diff
@@ -630,6 +630,7 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
   }
 
   sInstance = this;
+  Qgis::ogrRuntimeSupport();
```

Comment:
How exactly should the application be ended?
Which console error function should be used?